pipelines/base/enterprise-contract.yaml

```diff
@@ -0,0 +1,57 @@
+# The purpose of this pipeline is to execute the verify-enterprise-contract-v2 task for container
+# images that are built but not automatically released in order to provide early feedback to users.
+# When auto release is enabled, the task is executed by the release pipeline immediately after the
+# container images are built, thus it is not necessary to execute the task via this pipeline.
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: enterprise-contract
+  labels:
+    "skip-hacbs-test": "true"
+spec:
+  params:
+    - name: IMAGES
+      type: string
+      description: |
+        Spec section of an ApplicationSnapshot. The containerImage of each
+        component will be verified.
+    - name: POLICY_CONFIGURATION
```

Comment:
Is this something that the Integraton service will be able to figure out based on the Application's configuration or should we use the default in most cases?

Reply:
Good question. The Policy Configuration looks like this:
```
apiVersion: appstudio.redhat.com/v1alpha1
kind: EnterpriseContractPolicy
metadata:
  name: ec-policy
spec:
  description: Red Hat's enterprise requirements
  exceptions:
    nonBlocking: []
  publicKey: |
    -----BEGIN PUBLIC KEY-----
    ...
    -----END PUBLIC KEY-----
  rekorUrl: https://my.rekor.example.com
  sources:
  - git:
      repository: https://github.com/hacbs-contract/ec-policies.git
      revision: main
```

For the RedHat org, I believe all the fields will be the same. Someone, however, may want to add more policy repos under `.spec.sources` for additional validations.

For now, I think it's safe to assume that it'll always be the same ECP resource, but it's probably a good idea to plan for it to be configurable in the future (by whatever mechanism).

Wdyt?

Reply:
Ok, so if I understand correctly we should look for the ECP resource in the application's workspace and add that under this parameter?